docs/reference/commands/setup-passwords.asciidoc

```diff
@@ -4,7 +4,7 @@
 == elasticsearch-setup-passwords
 
 The `elasticsearch-setup-passwords` command sets the passwords for the built-in
-`elastic`, `kibana`, `logstash_system`, and `beats_system` users.
+`elastic`, `kibana`, `logstash_system`, `beats_system`, and `apm_server_system` users.
```

Comment:
Is it possible to call the user `apm-server_system`? I mainly worry that users might mistype it.

Reply:
Wouldn't mixing the dash and the underscore make it more easy for users to mistype the username?

Reply:
I think it could be argued both ways and probably both options are going to cause trouble. Go with the option you feel more comfortable with.

Reply:
@graphaelli Why don't you break the tie here, since this username is intended for APM customers after all 😄

Reply:
`apm_server_system` looks great, thanks